src/components/Resource/Resource.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+import { ListItem } from '@rneui/themed';
+import { styles } from './styles';
+
+interface SeekHelpProps {
+  org_name: string;
+  summary: string;
+  address: string;
+  phone_number: string;
+  hotline_number: string;
+  office_hours: string;
+  tags: string;
+}
+
+export default function ResourceT({
+  org_name,
+  summary,
+  address,
+  phone_number,
+  office_hours,
+}: SeekHelpProps) {
+  const [expanded, setExpanded] = useState(false);
+
+  return (
+    <View style={styles.resourceCard}>
+      <ListItem.Accordion
+        content={
+          <View style={styles.headerContent}>
+            <Text style={styles.orgName}>{org_name}</Text>
+            {!expanded && (
+              <Text style={styles.summaryPreview}>
+                {summary.slice(0, 60)}...
```

Comment:
ik u couldn't check during MPH but can you try to fix the caret on your iPad? It should be all the way on the right. Thanks!

Reply:
i checked on my iPad and the caret is all the way on the right

Reply:
wait jk i fixed it properly